static/js/lib/validation.js

```diff
@@ -145,6 +150,11 @@ export const validateEmailForm = validate([
     R.lensPath(["value", "email"]),
     "Email is not formatted correctly"
   ),
+  validation(
```

Comment:
Should move this above `validEmail` I think. No sense testing an invalid one for if it's an MIT one